Sources/TokamakDOM/DOMRenderer.swift

```diff
@@ -71,7 +71,7 @@ func appendRootStyle(_ rootNode: JSObject) {
 }
 
 final class DOMRenderer: Renderer {
-  private(set) var reconciler: StackReconciler<DOMRenderer>?
+  private var reconciler: StackReconciler<DOMRenderer>?
```

Comment:
Still keeping the property implementation itself around, as we need a non-zero retain count on it. The reconciler is really what drives the rendering process.